sass/_navigation.scss

```diff
@@ -0,0 +1,97 @@
+.bg__color--blue {
+	background-color: $blueish;
+
+	.body__container--nav {
+		max-width: 1300px;
```

Comment:
@Jelena-ag projekt, z tego co pamiętam, ma w założeniach napisane 2 breakpointy: 1440px i 375px, czyli tu max-width chyba by potrzeba ustawić 1440px?

Reply:
na logikę tak, niby ma być 1440px, ale jak tyle jest, to wisualnie mi nie pasuje do projektu. 
Zmienię na 1440px i dodam większe paddingi po bokach.